lib/cdo/geocoder.rb

```diff
@@ -164,6 +138,20 @@ def search(query, options = {})
     end
   end
   singleton_class.prepend SauceLabsOverride
+
+  # Override Geocoder#search to default to the RD country code for requests coming from the same box. These requests
```

Comment:
why do we want to do this?

Reply:
also, why the Dominican Republic?

Reply:
Because that was the existing behavior of the IP lookup service we use. Also, Dominican Republic appears to be `DO`? `RD` is not assigned to anything https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2

I could change it to something completely different like "LOOPBACK". I have to change the places in our codebase which check for `RD`.

Reply:
Here is an example of the code we have in a few places
```
-# geocoder sometimes shows localhost's country as RD/Reserved
- us_ip = location.nil? || ['US', 'RD'].include?(location.country_code.to_s.upcase)
```

Reply:
oooh, that makes more sense.

I'd love to see the comment updated to explain all that!

Reply:
`RD` / `Reserved` is a hard-coded default result for loopback addresses (e.g., `127.0.0.1`) that `geocoder` adds to [`Freegeoip` lookups](https://github.com/alexreisner/geocoder/blob/350cf0cc6a158d510aec3d91594d9b5718f877a9/lib/geocoder/lookups/freegeoip.rb#L41-L54).